### PR TITLE
chore(types): add types

### PR DESCRIPTION
**Pull request by eesteban**
Hi,

Would it be possible to add the types to the project :smile: ?

Thank you.

### CODE REVIEW

File: index.d.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+export interface IOptions {
+  header: any;
+};
+
+/**
+ * Decode jwt
+ * @param token
+ * @param key
+ * @param noVerify
+ * @param algorithm default to HS256
```
Comment (alexjab) [inline index.d.ts]: Why add the jsdoc here since it's already in the code?

Comment (eesteban) [reply index.d.ts]: You are right, it's not necessary :+1:  I will bump the version to 0.5.2 as I would like to make a feat request for the 0.6.0 in a PR to come.

**Comment by alexjab** [comment]
@paulsouche Done (sorry, I was AFK yesterday).
@hokaccha Can you `npm publish` ?

**Comment by hokaccha** [comment]
@paulsouche @alexjab Many thanks, I published it.

**Pull request merged**